.github/workflows/submodule_update.yml

```diff
@@ -0,0 +1,32 @@
+name: Submodule_update
+
+on:
+  push:
+    branches: 
+      - 'master'
+      - '2.10'
+
+jobs:
+  tarantool:
+    name: Update submodule in ee repository
+    runs-on: ubuntu-20.04-self-hosted
+    env:
+      SUBMODULE: 'tarantool'
+      REPOSITORY: "${{ secrets.EE_UPDATE_SUBMODULE_REPO }}"
+      FEATURE_BRANCH: 'TarantoolBot/update-submodule'
+      CHECKOUT_BRANCH: 'master'
+      PR_AGAINST_BRANCH: 'master'
```

Comment:
This is wrong. It should be `${{ github.ref_name }}`. Not `head_ref`.

Reply:
ok